input/CIB_Lenz.yml

```diff
@@ -0,0 +1,17 @@
+mapper_class: MapperCIBLenz
+map_name: CIB_Lenz
+file_map: /mnt/extraspace/damonge/Datasets/CIB_Lenz/cib_fullmission.hpx.fits
+file_hm1: /mnt/extraspace/damonge/Datasets/CIB_Lenz/cib_evenring.hpx.fits
+file_hm2: /mnt/extraspace/damonge/Datasets/CIB_Lenz/cib_oddring.hpx.fits
+file_mask: /mnt/extraspace/damonge/Datasets/CIB_Lenz/mask_apod.hpx.fits
+gal_mask_mode: '0.2'
```

Comment:
Humm, why do we need this? Isn't the mask defined in a single healpix map?

Reply:
You are right, mind fart

Reply:
you are right! corrected!